### PR TITLE
Automatically generated tsd files

### PR DESCRIPTION
**Pull request by jimlloyd**
@mhfrantz This adds a script that can be run to update DefinitelyTyped/ts-tinkerpop. The script is set to run as a postpublish hook.

I'm preparing a PR for DefinitelyTyped with contents that were automatically generated by the script.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,12 +15,13 @@
...
     "node-java"
   ],
-  "author": "Jim Lloyd <jim.lloyd@gmail.com>",
+  "author": "Jim Lloyd <jim.lloyd@gmail.com> (https://github.com/jimlloyd)",
   "license": "MIT",
   "devDependencies": {
     "chai": "^2.0.0",
     "dts-bundle": "^0.2.0",
     "groc": "^0.8.0",
+    "hb-interpolate": "^1.0.0",
```
Comment (mhfrantz) [inline package.json]: hb-interpolate == cool

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "lib/ts-tinkerpop.js",
   "scripts": {
     "test": "make test",
-    "install": "make install-maven"
+    "install": "make install-maven",
+    "postpublish": "./devbin/update-definitely-typed.sh"
```
Comment (mhfrantz) [inline package.json]: I'm not sure this makes complete sense as a postpublish.  There are times when you want to update TSD on a topic branch to do integration testing, and you would want to use ts-tinkerpop from a URL (pre-publish).  But we can leave it for now.

**Comment by mhfrantz** [comment]
LGTM

File: package.json (hunk 2 above)
Comment (jimlloyd) [inline package.json]: If you pre-publish, then when the postpublish script runs it should be a no-op (assuming you bumped the version number proactively).

The main reason I want it as a postpublish script is just so there is an automatic reminder to do that step. If the step is already done, the reminder is unnecessary, but doesn't hurt anything.

**Pull request merged**